README.md

```diff
@@ -1,24 +1,56 @@
+
+[![Waffle.io - Columns and their card count](https://badge.waffle.io/dylhof/family-stories.svg?columns=all)](https://waffle.io/dylhof/family-stories)
+
 # README
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+Family Stories is our cross-pollination group project for Module 4 of 4 at the Turing School of Software & Design.
+
+This repository is backend only and includes the following tech: Ruby on Rails & PostgreSQL.  Our frontend can be found at https://github.com/dylhof/family-stories.
+
+#### [**_View Family Stories in Production_**](https://family-secrets.herokuapp.com/)
+
+#### [**_View the backend data for Family Stories_**](https://sheltered-badlands-12986.herokuapp.com/api/v1/families)
+
+## Getting Started
+
+To run Family Stories on your local machine, navigate to the directory in which you would like the project to be located, then execute the following commands:
 
-Things you may want to cover:
+```
+$ git clone git@github.com:le3ah/sweata_weatha.git
+$ cd family_stories
+$ bundle
+$ rails g rspec:install
+$ rails db:create
+$ rails db:migrate
+$ rails db:seed
+```
 
```

Comment:
I think we should include a line on how to start up the server to view in localhost